distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -149,8 +149,8 @@ log4j2.appender.stdout.layout.pattern = ${log4j2.pattern}
 # out
 log4j2.appender.out.type = RollingFile
 log4j2.appender.out.name = out
-log4j2.appender.out.fileName = ${karaf.data}/log/${org.codice.ddf.system.branding}.log
-log4j2.appender.out.filePattern = ${karaf.data}/log/${org.codice.ddf.system.branding}.log-%d{yyyy-MM-dd-HH}-%i.log.gz
+log4j2.appender.out.fileName = ${karaf.log}/${org.codice.ddf.system.branding}.log
```

Comment:
❓ Confirming that this property is defined to point to the `data/log` directory and not elsewhere. If it does point elsewhere, we'll need to update docs.

Reply:
Yes it's pointing to the `data/log` directory by default. It's just an additional property they added to simplify configuration. I just stole it from their pax-logging configuration https://github.com/apache/karaf/commit/8c3dda195674a2c6424be070bbba066211ab02c8#diff-ca0709178d036c5c24035e21d4cbb32cR71